README.md

```diff
@@ -3,6 +3,10 @@
 ## Project Description:
 Our idea is to add a user-friendly way to cycle through tabs in the same window in Google Chrome, by order of the most recently opened tab. This should allow us to access our previously visited tab regardless of its location within other tabs with a simple click of the keyboard commands CTRL+Q. Additionally, if the user holds down CTRL and then presses Q however many times they want to press Q, the extension does the same thing as before and switches to the appropriate tab, but also displays a mini tab window pop up. This is similar to how the Windows OS’s alt-tab feature works. We also want to include options for customization such as changing keybindings and retaining the original functionality.
 
+## Getting started/Contributing
+How to build our project, run it, and test it? 
+See our [wiki](https://github.com/paulstn/SmarterTabSwitching/wiki) for information.
```

Comment:
nit, would be nice to include
```suggestion
See our [wiki](https://github.com/paulstn/SmarterTabSwitching/wiki) for information.
Report bugs and other issues using the "Issues tab". Make sure to check existing issues to avoid duplicates
```